NOTICE

```diff
@@ -1,11 +1,25 @@
-CRATE Active Record Adapter
-Copyright 2013-2015 CRATE Technology GmbH ("Crate")
+CrateDB Active Record Adapter
+Copyright 2013-2018 Crate.IO GmbH ("Crate")
 
 
-=========================================================================
+Licensed to Crate.IO GmbH (referred to in this notice as "Crate") under one or
+more contributor license agreements.
 
-Copyright (2014) by Christoph Klocker (http://www.vedanova.com)
-Special thanks go to Christoph for launching and writing the
-main parts of this project.
```

Comment:
@jodok please specifically review this removal

Reply:
i will talk to the original author.